jupyter/nmm-cache.ipynb

```diff
@@ -94,6 +98,8 @@
    "outputs": [],
    "source": [
     "def write(self, state, player: str, limit: int, value: float, alpha: float, beta: float) -> None:\n",
+    "    if limit < self.min_limit:\n",
```

Comment:
Bitte zur Doku hinzufügen, dass der `state` nur in den Cache aufgenommen wird, wenn das `limit` das `min_limit` überschreitet.

Reply:
Bzw. ist das überhaupt sinnvoll? Der Cache wird ja nach jedem `bestMoves()` gecleant? Dann können wir zwar die Cachegröße innerhalb des `bestMoves()`-Aufrufes überschreiten, aber das machen wir ja sowieso. Damit sparen wir uns dann aber trotzdem weiterhin sehr viele Aufrufe.

Reply:
Sonst wird der ja riesig...

Reply:
Der Cache ist dann nur Zeitweise (im RAM) größer, und das auch nur um maximal 25_000 Einträge, weil das ja das `max_states` Limit ist... Also so wirklich viel größer wird er dadurch nicht, macht aber die `bestMoves()` Funktion effizienter..

Reply:
Habs raus geschmissen.